tests/apps/runtime/app.go

```diff
@@ -107,6 +113,27 @@ func invokeDaprGRPCAPI() error {
 	return nil
 }
 
+func invokeStateStoreGetAPI() error {
+	stateStoreURL := fmt.Sprintf("http://%s/v1.0/state/%s/test", daprHTTPAddr, stateStoreName)
+	// nolint: gosec
+	r, err := http.Get(stateStoreURL)
+	if err != nil {
+		return err
+	}
+	defer r.Body.Close()
+
+	bytes, err := io.ReadAll(r.Body)
+	if err != nil {
+		return err
+	}
+
+	if r.StatusCode < 200 || r.StatusCode >= 400 {
```

Comment:
Do we have scenarios with status code < 200?

Reply:
https://docs.dapr.io/reference/api/state_api/#http-response-1
I expect 200 or 204, both of them are okay.